moveit_commander/src/moveit_commander/planning_scene_interface.py

```diff
@@ -44,8 +44,8 @@
 try:
     from pyassimp import pyassimp
 except:
-    # support pyassimp > 3.0
-    import pyassimp
```

Comment:
instead of removing this ``import``, can we leave it as a comment is its more obvious how to restore this functionality if/when pyassimp is fixed?